Description: Use explicit /lib for systemd service
 ${CMAKE_INSTALL_LIBDIR} will be arch-specific and something like
 lib/aarch64-linux-gnu/ which is not compatible with installing a systemd unit.
 Use the explicit lib/ path to avoid arch-specific path.
Author: Agathe Porte <agathe.porte@oss.qualcomm.com>
Last-Update: 2026-07-15

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -62,5 +62,5 @@
 # Install service
 pkg_check_modules(PC_SYSTEMD QUIET systemd)
 if(PC_SYSTEMD_FOUND)
-  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/urm.service DESTINATION ${CMAKE_INSTALL_LIBDIR}/systemd/system/)
+  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/urm.service DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/systemd/system/)
 endif()
